pyron-app/Dockerfile

```diff
@@ -39,10 +39,8 @@ ENV GC_PRINT_OPTS="-XX:+PrintGCDetails -XX:+PrintGCDateStamps \
 	-Dcom.sun.management.jmxremote.rmi.port=$JMX_PORT" \
 	NETWORK_OPTS=-Djava.net.preferIPv4Stack=true
 
-RUN addgroup -S cloudentity && \
-	adduser -S -G cloudentity cloudentity \
-	&& apk add --no-cache \
-	su-exec=${SU_EXEC_VERSION} curl=${CURL_VERSION}
+RUN groupadd -r cloudentity && useradd -u 9999 \
+	--no-log-init -r -g cloudentity cloudentity
```

Comment:
What do you think about putting extended option names in bash code? For me personally it is easier to understand and can be considered as clean code. For example:
```
groupadd --system cloudentity && useradd --uid 9999 \
  --no-log-init --system --gid cloudentity cloudentity
```
Maybe this particular example isn't so hard, but I generally prefer to use `--option-name` instead of `-O` wherever possible.